pkg/virt-handler/virtwrap/manager.go

```diff
@@ -203,19 +224,28 @@ func (l *LibvirtDomainManager) SyncVM(vm *v1.VM) (*api.DomainSpec, error) {
 		logging.DefaultLogger().Object(vm).Error().Reason(err).Msg("Getting the domain state failed.")
 		return nil, err
 	}
+
+	// To make sure, that we set the right qemu wrapper arguments,
+	// we update the domain XML whenever a VM was already defined but not running
+	if !newDomain && cli.IsDown(domState) {
+		_, err = l.setDomainXML(vm, wantedSpec)
```

Comment:
do we need to update the dom object here if we're redefining the domain?

Reply:
good question, will check that ...